tests/e2e_tests/test_github_app.py

```
@@ -94,5 +94,84 @@ def test_e2e_run_github_app():
+        # Create a pull request
+        logger.info(f"Creating a pull request from {new_branch} to {base_branch}")
+        pr = repo.create_pull(
+            title=new_branch,
+            body="update cli_pip.py",
+            head=new_branch,
+            base=base_branch
+        )
+
+        # check every 1 minute, for 5, minutes if the PR has all the tool results
+        for i in range(NUM_MINUTES):
+            logger.info(f"Waiting for the PR to get all the tool results...")
```

Ruff (0.8.2)

f-string without any placeholders

(F541)